src/chainparams.cpp

```diff
@@ -254,7 +254,7 @@ class CTestNetParams : public CChainParams {
 class CRegTestParams : public CChainParams {
 public:
     explicit CRegTestParams(const ArgsManager& args) {
-        strNetworkID = "regtest";
+        strNetworkID =  CBaseChainParams::REGTEST;
```

Comment:
There seems to be an unnecessary white space here.

Reply:
You may install `clang-format` and run the https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy script to preempt whitespace nitpicking.